R/pkg/R/serialize.R

```diff
@@ -36,15 +36,15 @@
 # nolint end
 
 getSerdeType <- function(object) {
-  type <- class(object)[[1]]
-  if (is.atomic(object) & !is.raw(object) & length(object) > 1) {
+  type <- class(object)[[1L]]
```

Comment:
btw why did we change 1 -> 1L? Is it related with the fix?

Reply:
no, just a microoptimization while i'm in this source